docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -650,9 +650,6 @@ AllCops:
   TargetRubyVersion: 2.5
 ----
 
-NOTE: When `ParserEngine: parser_prism` is specified, `TargetRubyVersion` must
```

Comment:
This is still true but I don't feel like it's worth to point out here anymore. Users aren't expected to configure this themselves anymore and it's also explained again below where `ParserEngine` is actually explained.